![video_game](video_game.jpg)


Video games are big business: the global gaming market is projected to be worth more than $300 billion by 2027 according to Mordor Intelligence. With so much money at stake, the major game publishers are hugely incentivized to create the next big hit. But are games getting better, or has the golden age of video games already passed?

In this project, you'll analyze video game critic and user scores as well as sales data for the top 400 video games released since 1977. You'll search for a golden age of video games by identifying release years that users and critics liked best, and you'll explore the business side of gaming by looking at game sales data.

Your search will involve joining datasets and comparing results with set theory. You'll also filter, group, and order data. Make sure you brush up on these skills before trying this project! The database contains two tables. Each table has been limited to 400 rows for this project, but you can find the complete dataset with over 13,000 games on Kaggle.

### `game_sales` table

| Column | Definition | Data Type |
|-|-|-|  
|name|Name of the video game|`varchar`|
|platform|Gaming platform|`varchar`|
|publisher|Game publisher|`varchar`|
|developer|Game developer|`varchar`|
|games_sold|Number of copies sold (millions)|`float`|
|year|Release year|`int`|

### `reviews` table

| Column | Definition | Data Type |
|-|-|-|
|name|Name of the video game|`varchar`|  
|critic_score|Critic score according to Metacritic|`float`|
|user_score|User score according to Metacritic|`float`|


### `users_avg_year_rating` table

| Column | Definition | Data Type |
|-|-|-|
|year| Release year of the games reviewed |`int`|  
|num_games| Number of games released that year |`int`|
|avg_user_score| Average score of all the games ratings for the year |`float`|

### `critics_avg_year_rating` table

| Column | Definition | Data Type |
|-|-|-|
|year| Release year of the games reviewed |`int`|  
|num_games| Number of games released that year |`int`|
|avg_critic_score| Average score of all the games ratings for the year |`float`|



# Objetivo General
Analizar la evolución de la industria de los videojuegos desde 1977 hasta la actualidad a través de datos de ventas y calificaciones de usuarios y críticos, con el fin de identificar patrones de éxito, tendencias de calidad y posibles “edades doradas” del gaming.


## ¿Cuáles son los videojuegos más exitosos comercialmente?

In [4]:
SELECT name , year , games_sold from game_sales
order by games_sold desc
limit 10

name  year  games_sold
0                         Wii Sports for Wii  2006       82.90
1                  Super Mario Bros. for NES  1985       40.24
2    Counter-Strike: Global Offensive for PC  2012       40.00
3                     Mario Kart Wii for Wii  2008       37.32
4       PLAYERUNKNOWN'S BATTLEGROUNDS for PC  2017       36.60
5                           Minecraft for PC  2010       33.15
6                  Wii Sports Resort for Wii  2009       33.13
7  Pokemon Red / Green / Blue Version for GB  1998       31.38
8               New Super Mario Bros. for DS  2006       30.80
9          New Super Mario Bros. Wii for Wii  2009       30.30

## ¿Cuáles son los 10 años con mayor puntuación media de la crítica y muchos éxitos producidos?

In [5]:
SELECT * from critics_avg_year_rating
order by avg_critic_score desc
limit 10

year  num_games  avg_critic_score
0  1998         10              9.32
1  2004         11              9.03
2  2002          9              8.99
3  1999         11              8.93
4  2001         13              8.82
5  2011         26              8.76
6  2016         13              8.67
7  2013         18              8.66
8  2008         20              8.63
9  2017         13              8.62

el mejor año para los videojuegos segun sus criticas fue en 1998 

## ¿Cuáles son los 10 años con mayor puntuación media de los usuarios y mayor número de visitas producidas?


In [7]:
SELECT * from users_avg_year_rating
order by avg_user_score desc , num_games desc
limit 10

year  num_games  avg_user_score
0  1997          8            9.50
1  1998         10            9.40
2  2010         23            9.24
3  2009         20            9.18
4  2008         20            9.03
5  1996          5            9.00
6  2006         16            8.95
7  2005         13            8.95
8  1999         11            8.80
9  2002          9            8.80

el mejor año segun los usuarios tambien fue en 1997

## ¿Qué años tuvieron las puntuaciones medias más altas tanto de críticos como de jugadores y muchos éxitos producidos?

In [10]:
SELECT  c.year, c.avg_critic_score, u.avg_user_score, c.num_games AS critic_games, u.num_games AS user_games
FROM critics_avg_year_rating c
INNER JOIN users_avg_year_rating u 
    ON c.year = u.year
order by c.avg_critic_score desc , u.avg_user_score desc
limit 10

year  avg_critic_score  avg_user_score  critic_games  user_games
0  1998              9.32            9.40            10          10
1  2004              9.03            8.55            11          11
2  2002              8.99            8.80             9           9
3  1999              8.93            8.80            11          11
4  2001              8.82            7.23            13          13
5  2011              8.76            8.21            26          26
6  2016              8.67            6.95            13          13
7  2013              8.66            8.22            18          18
8  2008              8.63            9.03            20          20
9  2012              8.62            7.31            12          12

Segun critica y usuarios el mejor año fue 1998

## ¿Cuál es el número de juegos vendidos en esos años clave?


In [3]:
SELECT SUM(gs.games_sold) AS total_sales, gs.year
FROM game_sales gs
WHERE gs.year IN (1998,2004,1999,2001,2016,2013,2008,2012)
GROUP BY gs.year
ORDER BY total_sales DESC;

total_sales  year
0       177.23  2013
1       175.07  2008
2       161.30  2012
3       113.02  2004
4       105.01  2016
5       101.52  1998
6        95.28  2001
7        74.90  1999

Vemos que el año donde se vendieron mayor cantidad de juegos fue el 2013 como 177 millones de videojuegos